runtime/acala/src/lib.rs

```diff
@@ -1820,27 +1828,33 @@ impl_runtime_apis! {
 			gas_limit: u64,
 			storage_limit: u32,
 			access_list: Option<Vec<AccessListItem>>,
-			estimate: bool,
+			rpc_mode: bool,
```

Comment:
is this backward compatible?

Reply:
what do you mean?

Reply:
this is runtime API, so two thing will happen:
client upgraded before runtime upgrade, so new client + old runtime
runtime upgraded before client upgrade, so old client + new runtime

in this case the runtime API remains the same so at lease it won't be crashing, but does the behaviour still remains reasonable when runtime and client are on different version?

Reply:
@xlc this will require new client + new runtime

Reply:
we are not going to be able to upgrade both runtime & client at same time, so we need to have some versioning

but in this particular case, will it be any incompatibility with old client + new runtime? the function signature is same and the semantic is still somewhat compatible?